packages/parchment/test/convert/html_test.dart

```diff
@@ -256,11 +256,10 @@ void main() {
 
         expect(
           codec.encode(doc.toDelta()),
-          '<pre>'
-          'void main() {\n'
-          '\n'
-          '  print("Hello World!");\n'
-          '}\n'
+          '<pre><code>void main() {</code>\n'
```

Comment:
There might be some misunderstanding
Your previous implementation was correct only it needed the whole block to be `<pre><code>` and only the block
```html
<pre><code>
void main() {
 
  println(...)
}
</code></pre>
```

and **not**

```html
<pre>
<code>void main() {</code>
<code>\n</code>
<code>  println(...)</code>
<code>}</code>
</pre>
```

Reply:
Changed.